packages/manager/cypress/e2e/core/linodes/plan-selection.spec.ts

```
@@ -124,7 +124,7 @@ const notices = {
 
 authenticate();
 describe('displays linode plans panel based on availability', () => {
-  before(() => {
+  beforeEach(() => {
```

Using `before` here instead of `beforeEach` causes the mocks not to get applied on re-attempts which leads to failures.

The initial failure is caused by the `cy.click()` issue, but `plan-selection.spec.ts` failures are overrepresented in CI due to the re-attempts being guaranteed to fail.